TROUBLESHOOTING.md

```diff
@@ -54,16 +122,15 @@ Instead of loading all tools, configure targeted MCP servers for your specific n
 ```
 *Result: ~15-20 tools total instead of 128+*
 
-*Available Azure Services for `--service` flag:*
+*Available Azure Services:*
 `storage`, `keyvault`, `cosmos`, `redis`, `servicebus`, `monitor`, `appconfig`, `kusto`, `postgres`, `search`
```

Comment:
We'll need to keep updating this list. Are there other places in docs where this also happens? Maybe there's a way we can mark them and automate that or at least have some CI check